### ISSUE TITLE
Dungeon Protection Issue

### ISSUE THREAD

**Issue by Oscarmike06** [TeamChocoQuest/ChocolateQuestRepoured#248]
**Common sense Info**
 - I play...
   - [X] With a large modpack
   - [ ] Only with CQR and it's dependencies
 - The issue occurs in...
   - [ ] Singleplayer
   - [X] Multiplayer
 - [X] I have searched for this or a similar issue before reporting and it was either (1) not previously reported, or (2) previously fixed  and I'm having the same problem.
 - [X] I am using the latest version of the mod (all versions can be found on github under releases)
 - [X] I read through the FAQ and i could not find something helpful ([FAQ](https://wiki.cq-repoured.net/index.php?title=FAQ))
 - [ ] I reproduced the bug without any other mod's except forge, cqr and it's dependencies
 - [ ] The game crashes because of this bug
 
 
 **Versions**
Chocolate Quest Repoured ('latest' IS NOT a valid version! Please use the version string that shows up in the mod overview of forge): 2.2.3B
Forge:14.23.5.2854
Minecraft:1.12.2

**Describe the bug**
I have cleared two dungeons and killed the boss, after that I am able to break blocks, however when I leave the chunk and return the protection has restarted. 

**Expected behavior**
Once the boss has died, the protection system does not restart.

**Comment by DerToaster98** [comment]
Can't reproduce it in singleplayer, tried in multiplayer: Killing the boss didn't disable the protection zone.

For a workaround you can try to disable the protection system and to delete all protected region files (that IS explained in the faq, i won't explain it here)

**Comment by DerToaster98** [comment]
However, this does seem to be caused randomly cause i tried it a few times and in 9 out of 10 cases i could break the blocks after revisiting the dungeon. I guess you are just using an old version (the one in the modpack you use would be outdated then, or the version on the server is an old one)

**Comment by Oscarmike06** [comment]
The version I am using is the only one that is on curse forge.
![image](https://user-images.githubusercontent.com/78283762/106397216-d18e2300-63d1-11eb-9865-02de5a4b7962.png)

I have looked for a "force field nexus" on the two that I killed the boss on and could not find one. 

I have read the workaround but it did seem that it referred to the alpha version, so I wanted to make sure it was still used if needed. Removing the folder will that disable protection for all that have been generated or will it turn it off for good?
